amm/src/main/scala/ammonite/interp/ImportHook.scala

```diff
@@ -108,7 +108,7 @@ object ImportHook{
                 ))
 
                 Result.Source(
-                  read(filePath),
+                  read(filePath).replace("\r", "").replace("\n", System.lineSeparator()),
```

Comment:
We're doing a bunch of ad-hoc `.replace`s all over the place; let's aggregate them all into a helper function `normalizeNewlines(s: String)` and use that everywhere